src/store/appoint.js

```diff
@@ -0,0 +1,45 @@
+import Axios from "axios";
```

Comment:
Please rename store `appoints` to  `appointments`

Reply:
done